dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1125,6 +1187,77 @@
         }
       }
     },
+    "EntityComponentInformation": {
+      "description": "The object for entity component information.",
+      "type": "object",
+      "discriminator": "entityComponentKind",
+      "properties": {
+        "entityComponentKind": {
```

Comment:
Thank you for disambiguating the type, but I would offer a compromise here: normally, discriminators for the REST APIs might just be generic like "kind" here; but, to disambiguate in an SDK where multiple services / SDKs may have such a generic term, that's where you disambiguate. Thus, you'd set `x-ms-enum.name` to "entityComponentKind".